source/whats-new.txt

```diff
@@ -44,6 +45,40 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.2:
+
+What's New in 6.2
+-----------------
+
+The {+driver-short+} v6.2 release includes the following features:
+
+- Updates the ``bson`` package version to 6.2.0. This version includes
+  color visualization of types, as shown in the following image:
+
+  .. figure:: /includes/figures/bson-color.png
+     :alt: A screenshot of the terminal that shows printing in color
+
+  To learn more, see the `v6.2.0 bson release notes
+  <https://github.com/mongodb/js-bson/releases/tag/v6.2.0>`__.
+
+- Ensures that the ``result.insertedIds`` property of a bulk write error type
+  contains the ``_id`` values of successfully inserted documents. In
+  previous versions, even if a bulk write operation rejects any insert
+  operation, the ``result.insertedIds`` property still contains the
+  ``_id`` of each attempted insert.
+
+- Closes the implicit session created when running the ``findOne()``
+  method on a time series collection regardless of the outcome of the
+  find operation.
```

Comment:
Suggestion:

Since there's only one operation mentioned, I think omitting the more general classifier "find" could make it less likely to generate questions as to whether there are other find operations to which this applies.

```suggestion
  operation.
```